methods.py

```diff
@@ -0,0 +1,23 @@
+class MethodsOfPython:
+
+    def __init__(self):
+        print("instance method")
+
+    @classmethod
+    def class_method(cls):
+        print("class method1")
+
+    @staticmethod
+    def static_method():
+        print("static method")
+
+    def inst_methond(self):
+        print("calling instance method")
+
+
+MethodsOfPython.static_method()
```

Comment:
Add __name__ == "__main__" to run this code.

Reply:
Updated